src/app/home/home.component.ts

```diff
@@ -35,6 +35,9 @@ export class HomeComponent implements OnInit {
         console.log(this.loginForm.value);
          sessionStorage.setItem('access_token', '234');
               sessionStorage.setItem('user_id', '123');
+
+              localStorage.setItem('access_token', '234');
```

Comment:
@Bithika-Redapple These keys should all be constants. You want one point of reference for these key values, which will then allow you to reuse. Update your code structure and architect to not have to hardcode any values, anywhere. Never hardcode.